account_payment_partner/views/account_payment_mode.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--
+  © 2017 Sergio Teruel <sergio.teruel@tecnativa.com>
+  License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+-->
+
+<odoo>
+<data>
```

Comment:
In version 10 "<data>" is only necessary when you use "noupdate". 
(For example: <data noupdate = "1">)
You can omit it in this case.